### DISCUSSION TITLE
参考文献中某个会议超过列距，该怎么办？

### DISCUSSION THREAD

**Discussion by stella-zhou** [sjtug/SJTUThesis#932] [提问]
求助大佬，参考文献中只有[5]这个会议超过了列距，其他的都很正常，这应该怎么处理？
<img width="502" alt="截屏2023-04-22 15 37 03" src="https://user-images.githubusercontent.com/65953185/233772779-4f2abe31-ab89-4261-8e03-f4449ca2c0b8.png">

**Comment by note286** [comment]
参考 https://github.com/hushidong/biblatex-gb7714-2015/issues/11 。

**Reply by stella-zhou** [reply]
谢谢，这个要在哪个文件里面改，不是在bib里吧？
